lib/Subscriber/Behavior/Path/AutoNameSubscriber.php

```diff
@@ -7,8 +7,10 @@
  *
  * This source file is subject to the MIT license that is bundled
  * with this source code in the file LICENSE.
+ *
  */
 
+
```

Comment:
These two additions were not here before, were they? Both of them should be removed, same for the other file.